pom.xml

```diff
@@ -5,7 +5,7 @@
 
 	<groupId>org.springframework.data</groupId>
 	<artifactId>spring-data-redis</artifactId>
-	<version>1.8.12.BUILD-SNAPSHOT</version>
+	<version>1.8.12.DATAREDIS-623-SNAPSHOT</version>
```

Comment:
We keep adjusting version numbers in pull requests only when the branch is built by our CI to provide unique artifacts for branch builds. No need to include that type of change in future pull requests.